docs/workflows/aws.md

```diff
@@ -78,11 +79,14 @@ The AWS pipeline runs using a docker container. At runtime, the script [run_aws_
 - `burst_id_list` -> A list of burst id's corresponding to the scene. If not provided, all will be processed. Can be space separated list or line separated .txt file.
 - `resolution` -> The target resolution of the products. Default is 20m.
 - `output_crs` -> The target crs of the products. If not specified, the UTM of the scene center will be used. Expects integer values (e.g. 3031)
-- `dem` -> The dem to be used in processing. Supported is `cop_glo30`.
+- `dem_type` -> The dem to be used in processing. Supported is [`cop_glo30`, `REMA_32`, `REMA_10`, `REMA_2`].
 - `product` -> The product being created with the workflow. Must be `RTC_S1` or `RTC_S1_STATIC`.
 - `s3_bucket` -> the bucket to upload the products
 - `s3_project_folder` -> The project folder to upload to.
 - `collection` -> The collection which the set of products belongs.
+- `make_existing_products` -> If products should be made, even if they already exist in S3. 
+  - **WARNING**, - Setting will create duplicate files and overwrite metadata that will impact downstream processes.
```

Comment:
Suggest

Whether to generate products even if they already exist under the `s3_bucket/s3_project_folder/collection`.
 **WARNING** - Passing this flag will create duplicate files and overwrite existing metadata, which may affect downstream workflows.